compiler/luci/import/src/Importer.cpp

```diff
@@ -122,12 +122,19 @@ void convert_graph(const luci::GraphBuilderSource &source, luci::CircleReader &r
   }
 
   // Create CircleConst nodes for constant tensors.
+  // Create CircleNodes for constant tensors.
   // NOTE Origin is intentionally not provided for constants.
+  auto const_builder = source.lookup(luci::NodeBuilderType::BUFFER);
+  if (not const_builder)
+    throw oops::UserExn("Not supported", "tensor with buffer builder");
```

Comment:
Could you explain more about why this change is introduced?

Reply:
This change is introduced because `source.lookup()` returns nullptr in case is builder is not found. So, we need to check `nullptr` before using returned builder.

https://github.com/Samsung/ONE/blob/daea9fdc9f8a6338d14d2be88c14f7187256be93/compiler/luci/import/include/luci/Import/GraphBuilderRegistry.h#L37-L40

We also make this check with builder for operators:

https://github.com/Samsung/ONE/blob/daea9fdc9f8a6338d14d2be88c14f7187256be93/compiler/luci/import/src/Importer.cpp#L151
https://github.com/Samsung/ONE/blob/daea9fdc9f8a6338d14d2be88c14f7187256be93/compiler/luci/import/src/Importer.cpp#L170-L173